game/script.rpy

```diff
@@ -0,0 +1,38 @@
+﻿# The script of the game goes in this file.
+
+init python:
+    import src.TestModule
+
+# Declare characters used by this game. The color argument colorizes the
+# name of the character.
+
+define e = Character("Eileen")
+
+
+# The game starts here.
+
+label start:
+
+    # Show a background. This uses a placeholder by default, but you can
+    # add a file (named either "bg room.png" or "bg room.jpg") to the
+    # images directory to show it.
+
+    scene bg room
+
+    # This shows a character sprite. A placeholder is used, but you can
+    # replace it by adding a file named "eileen happy.png" to the images
+    # directory.
+
+    show eileen happy
+
+    # These display lines of dialogue.
+
+    e "You've created a new Ren'Py game."
+
+    e "Once you add a story, pictures, and music, you can release it to the world!"
+
+    $ e("Your random number is " + str(src.TestModule.get_random_number()) + "!")
```

Comment:
This is where I added the new method to the example scene. Unfortunately it looks like we need to include the src dir in the path to make use of it. We can probably get rid of it using an alias, or a different style import. We can play around with that more when we've got a few more files to play with.